assets/css/admin.scss

```diff
@@ -177,6 +177,36 @@
 	}
 }
 
+/**
+ * Notice info
+ */
+
```

Comment:
@roshansameer Why extra line is present here?

Reply:
There should be a line gap where there appears multiline comment. If there is single line comment there should not be a line gap. Ref ( Multi line comment ) : https://github.com/Automattic/_s/blob/master/sass/_normalize.scss#L22
Ref ( Single line comment ) : https://github.com/Automattic/_s/blob/28f0a5b4e56a5dc3cba94749a208eb478fe224d4/sass/mixins/_mixins-master.scss#L7

Reply:
@roshansameer Let's rule it and in other parts where they exist during file edit. This has been done by [normalize.css](http://necolas.github.io/normalize.css/) and I couldn't find any reason why it should be implemented.

Reply:
I have removed that :)